envoy/router/context.h

```diff
@@ -35,5 +35,23 @@ class Context {
   virtual GenericConnPoolFactory& genericConnPoolFactory() PURE;
 };
 
+/**
+ * RouteStatsContext defines config needed to generate all route level stats.
+ */
+class RouteStatsContext {
+public:
+  virtual ~RouteStatsContext() = default;
+
+  /**
+   * @return stat name for route specific stats.
+   */
+  virtual const Stats::StatName statName() const PURE;
```

Comment:
What does it mean to return a `const` of a value?

Reply:
yeah. it is not needed. Will remove